babel.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       '@babel/preset-env',
       {
         targets: {
+          browsers: ['ie >= 11'],
```

Comment:
Setting this allows storybook to render correctly on ie11 to avoid having to directly go to the component URL